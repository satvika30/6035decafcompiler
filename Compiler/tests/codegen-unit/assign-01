class Program {
	void main() {
		int a,b,c;
		a = 10;
		b = 15;
		c = 20;
		
		a += 5;
		callout("printf","%d\n", a);
		
		b -= a;
		callout("printf","%d\n", b);
		
		c += a + b;
		callout("printf","%d\n", c);
		
		b -= 11;
		callout("printf","%d\n", b);
	}
}