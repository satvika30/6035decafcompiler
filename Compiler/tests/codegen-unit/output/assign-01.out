15
0
35
-11
